services/ui-src/src/types/users.ts

```diff
@@ -25,8 +26,10 @@ export interface MFPUser {
 }
 
 export interface UserContextShape {
+  user: MFPUser | null | undefined;
```

Comment:
you could write this as
`user?: MFPUser`

then the question mark takes care of the `null | undefined` for you

Reply:
I'm not sure if it's because of the version of typescript we have? but when I swap out `MFPUser | null | undefined`, I get an error of `Type 'null' is not assignable to type 'MFPUser | undefined'.` when running `./dev local`